frontends/web/src/components/transactions/transactions.tsx

```diff
@@ -52,9 +53,11 @@ class Transactions extends Component<Props> {
                     <label className="labelXLarge">{t('accountSummary.transactionHistory')}</label>
                     { !csvExportDisabled && (
                         exported ? (
-                            <A href={exported} className="labelXLarge labelLink">{t('account.openFile')}</A>
+                            <A key="open" href="#" onClick={() => apiPost('open', exported)} className="labelXLarge labelLink">
```

Comment:
what's the purpose of the new `key` attributes? Was there a bug?

In any case, `A` is probably not the right tag anymore for this, right? Why use an Anchor when the logic of the Anchor component is not needed.  Could use `<a>` or `<span>` or sth. It is also confusing because Anchor.tsx already sets onClick by itself, and now also via the passed props. Maybe something a linter should catch or will catch in the future.

Reply:
No reason, just thougth it is best practice want me to remove?

kept using `<A>` and thuoght refactor in a different PR, see reason in commit msg:

> But kept the anchor component that the layout does not break.

Reply:
is it best practice? i thought it is only needed in lists so that rendering doesn't break, but maybe I missed sth.

Reply:
It doesn't seem to be a problem here, however it can conditionally render the same elements (`<a>`). Should I remove it because it works without or keep as I think it might be clearer with the key prop?